### PR TITLE
Add jitpack to instructions

### LINKED ISSUE #572: Put the latest version in the README
# Feature Request

## Description

I'm fairly new to Kotlin and Android development, so maybe this is a stupid issue, but it's frustrating that I want to use this library, but the installation instructions include `<package>:<latest-version>`, so I have to figure out where the actual versions are hidden. After about 5 minutes of navigating around the repo and documentation site, I've found the required Kotlin version, the required Coroutine version, and the latest legacy version, but I'm still clueless about what the current `2.x.y` version is.

I tried `implementation 'com.github.kittinunf.fuel:fuel:2.0.0'`, but apparently there is no version `2.0.0`. I also checked the releases page, but it lacks any `2.x.y` releases.

## Proposed Solution

Put the latest version in the install instructions or somewhere else in the README.

### PR DESCRIPTION
**Pull request by SleeplessByte**
Fixes #572, #566, #565, #555

Co-Authored-By: nikkyai <root@nikky.moe>

## Type of change

Check all that apply

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (a change which changes the current internal or external interface)
- [X] This change requires a documentation update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +51,50 @@ Each of the extensions / integrations has to be installed separately.
 | [`fuel-jackson`](./fuel-jackson) | _Deserialization_: [`Jackson`](https://github.com/FasterXML/jackson-module-kotlin)
 | [`fuel-moshi`](./fuel-moshi) | _Deserialization_: [`Moshi`](https://github.com/square/moshi)  |
 
+### Jitpack
+
+```kotlin
```
Comment (kittinunf) [inline README.md]: [Q] Should we care about our groovy user? 😺 😈

Comment (SleeplessByte) [reply README.md]: Groovy uses maven right? I don't mind having more instructions in here, but don't groovy users usually know how to do this :P ?

File: README.md [hunk 2]
```diff
@@ -48,6 +51,50 @@ Each of the extensions / integrations has to be installed separately.
...
+
+dependencies {
+  implementation(group = "com.github.kittinunf.fuel", name = "fuel", version = "-SNAPSHOT")
+  implementation(group = "com.github.kittinunf.fuel", name = "fuel-coroutines", version = "-SNAPSHOT")
+  implementation(group = "com.github.kittinunf.fuel", name = "fuel-kotlinx-serialization", version = "-SNAPSHOT")
+}
+```
+
+
+```kotlin
```
Comment (NikkyAI) [inline README.md]: this is a example of how to make the configuration more compact, maybe this should be mentioned explicitly

File: README.md (hunk 1 above)
Comment (NikkyAI) [reply README.md]: groovy users should know how to do the same thing ..if this turns out to be a problem we can still look for how to make dual language codeblocks.. like on the gradle docs

**Pull request merged**